UseCases/Sources/HTTPEndToEnd/Services/OrderServiceHandler.swift

```diff
@@ -0,0 +1,49 @@
+import AsyncHTTPClient
+import Baggage
+import BaggageLogging
+import Instrumentation
+import Logging
+import NIO
+import NIOHTTP1
+
+final class OrderServiceHandler: ChannelInboundHandler {
+    typealias InboundIn = HTTPServerRequestPart
+    typealias OutboundOut = HTTPServerResponsePart
+
+    private let httpClient: InstrumentedHTTPClient
+    private let instrument: Instrument<HTTPHeaders, HTTPHeaders>
+    private let logger = Logger(label: "OrderService")
+
+    init<I>(httpClient: InstrumentedHTTPClient, instrument: I)
+        where I: InstrumentProtocol,
+        I.InjectInto == HTTPHeaders,
+        I.ExtractFrom == HTTPHeaders {
+        self.httpClient = httpClient
+        self.instrument = Instrument(instrument)
+    }
+
+    func channelRead(context: ChannelHandlerContext, data: NIOAny) {
+        guard case .head(let requestHead) = self.unwrapInboundIn(data) else { return }
+
+        var baggage = BaggageContext()
```

Comment:
thats actually an interesting case of why calling it context would have been annoying... here we'd very confusingly nameclash hm hm.

Looks good!